openforcefield/utils/toolkits.py

```diff
@@ -352,18 +354,22 @@ def from_object(self, object):
         ----------
         object : A molecule-like object
             An object to by type-checked.
+
         Returns
         -------
-        Molecule or False
-            An openforcefield.topology.molecule Molecule, or False if loading was unsuccessful
+        Molecule
```

Comment:
Yeah. `Molecule or False` was a terrible idea. I learned my lesson trying to work around that.